.gitignore

```diff
@@ -30,3 +30,4 @@
 .mindtagger.conf.json
 /out
 /log
+/lib
```

Comment:
But do any of our source or build produce `lib/` or `library/`?  I don't think we need to over ignore.